src/api/app/assets/stylesheets/webui/application.scss

```diff
@@ -33,7 +33,7 @@
 @import 'coderay';
 @import 'system-status';
 @import 'texts';
-@import 'build-results';
+@import 'build-result';
```

Comment:
Just wondering, why is it singular now? I would keep this plural.

Reply:
> What is the reason for the `DO NOT MERGE` label? Waiting on #10861?

The reason was explained in the description. I want to squash the commits before merging and I was waiting for at least one more review before doing so (to make it easier for the reviewer).

Reply:
The file under `responsive_ux` namespace was singular. I thought that maybe was a reason behind that change. But it's true that I would also go for a plural name. As we both agree, I'm going to rename it. :+1: